src/bridge.cpp

```diff
@@ -3,33 +3,417 @@
 
 #include "bridge.h"
 
-#include "geometry/OpenLinesSet.h"
-#include "geometry/OpenPolyline.h"
+#include <range/v3/action/stable_sort.hpp>
+#include <spdlog/spdlog.h>
+#include <spdlog/stopwatch.h>
+
+#include "geometry/Point2D.h"
+#include "geometry/PointMatrix.h"
 #include "geometry/Polygon.h"
+#include "settings/EnumSettings.h"
 #include "settings/types/Ratio.h"
 #include "sliceDataStorage.h"
 #include "utils/AABB.h"
+#include "utils/linearAlg2D.h"
+#include "utils/math.h"
+
+// #define SVG_DEBUG 1
+
+#ifdef SVG_DEBUG
+#include "utils/SVG.h"
+#endif
```

Comment:
I think we should remove these before merging to main. I know this won't up in production code but I believe these kind of debugging statements do clutter the code-base, and I also notice I rarely use them since new debugging flows will require a slightly different debugging output most of the time.